### PR TITLE
fix(costmodels): cost models details fit and finish

### LINKED ISSUE #1048: Cost models fit & finish: details page
**Describe the bug**
Cost model details page fit and finish

**Expected behavior**
 - [ ] Delete dialog should be smaller
   image
 - [ ] If a source is assigned to a cost model the dialog button should be close and not cancel
   image
 - [ ] In the kebab option, delete should be last and its font color should be red.
   image

### PR DESCRIPTION
**Pull request by boaz0**
closes #1048 

 - [x] Delete dialog should be smaller
   
image

 - [x] If a source is assigned to a cost model the dialog button should be close and not cancel

image

 - [x] In the kebab option, delete should be last and its font color should be red.

image

### CODE REVIEW

**Comment by dayleparker** [comment]
Hi @boaz0,
I'm not sure if it's too late, but just saw this PR and spotted a couple small text edits:
* we need a full stop after 'undone' -->  "cannot be undone."
* Edits to "This action... cost model" sentence: "You must unassign any sources before you can delete this cost model." (Reword if you wish)

Please let me know if you want me to open a new issue instead, or if you'd rather me do a full text review later.
Thank you, otherwise all looks great! 👍

**Comment by boaz0** [comment]
@dayleparker - no need to open a ticket I can apply the changes in this PR.

**Comment by boaz0** [comment]
//ping @dayleparker 

image

image

File: src/pages/costModelsDetails/components/dialog.tsx [hunk 1]
```diff
@@ -45,22 +49,29 @@ const DialogBase: React.SFC<Props> = ({
...
   const actions =
     actionText !== ''
       ? [CancelButtonSecondary, ProceedButton]
-      : [CancelButtonPrimary];
+      : [CloseButtonPrimary];
   return (
-    <Modal title={title} isOpen={isOpen} onClose={onClose} actions={actions}>
+    <Modal
+      isLarge={isLarge}
+      isSmall={isSmall}
```
Comment (dlabrecq) [inline src/pages/costModelsDetails/components/dialog.tsx]: I see `isSmall` used in costModelsTable.tsx. Is `isLarge` still being used anywhere?

**Comment by dayleparker** [comment]
Looks great @boaz0! Apologies for not seeing your ping earlier. Thank you for making these changes.

File: src/pages/costModelsDetails/components/dialog.tsx (hunk 1 above)
Comment (boaz0) [reply src/pages/costModelsDetails/components/dialog.tsx]: No I will remove it.

**Pull request merged**